libfreerdp/core/input.c

```diff
@@ -218,37 +242,45 @@ void input_send_fastpath_unicode_keyboard_event(rdpInput* input, UINT16 flags, U
 
 	eventFlags |= (flags & KBD_FLAGS_RELEASE) ? FASTPATH_INPUT_KBDFLAGS_RELEASE : 0;
 	s = fastpath_input_pdu_init(rdp->fastpath, eventFlags, FASTPATH_INPUT_EVENT_UNICODE);
+	if (!s)
```

Comment:
fastpath_input_pdu_init_header returns a stream with 256 byte or FALSE. We only write 2 bytes plus headers here.